metricbeat/module/etcd/self/data.go

```diff
@@ -74,15 +74,15 @@ func eventMapping(content []byte) common.MapStr {
 			"appendrequest": common.MapStr{
 				"count": data.Recv.Appendrequest.Count,
 			},
-			"bandwithrate": data.Recv.Bandwithrate,
-			"pkgrate":      data.Recv.Pkgrate,
+			"bandwidthrate": data.Recv.Bandwidthrate,
```

Comment:
Good news is etcd is still in beat so we can do these breaking changes. But we must add it to the changelog.